src/shared/components/RadioButton/RadioButton.style.ts

```diff
@@ -18,44 +23,47 @@ export const Input = styled.input`
 const colorFromProps = ({ error, checked, disabled }: RadioButtonStyleProps) => {
```

Comment:
The active state seems to be missing the background

Figma:
<img width="478" alt="Screenshot 2021-04-08 at 11 33 59" src="https://user-images.githubusercontent.com/12646744/114003892-6db72900-985e-11eb-9663-621d288c2392.png">

App:
<img width="397" alt="Screenshot 2021-04-08 at 11 34 38" src="https://user-images.githubusercontent.com/12646744/114003922-73147380-985e-11eb-9949-0771f37bab8c.png">